src/components/expenses/expensesItem.tsx

```diff
@@ -0,0 +1,76 @@
+import {useNavigation} from '@react-navigation/native';
+import * as React from 'react';
```

Comment:
Import this like this:
`import React from 'react';`

And use on top of your component file.